Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -50,37 +175,79 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         finish()
     }
 
-    @objc func showHideAccuracyRadius() {
-        showsAccuracyRing.toggle()
+    private func addCustomizePuckButton() {
+        // Set up button to change the puck options
+        let button = UIButton(type: .system)
+        button.setTitle("Customize puck", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.backgroundColor = #colorLiteral(red: 0, green: 0.4784313725, blue: 0.9882352941, alpha: 1)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.layer.cornerRadius = 20
+        button.addTarget(self, action: #selector(changePuckOptions(sender:)), for: .touchUpInside)
+        view.addSubview(button)
+
+        // Set button location
+        NSLayoutConstraint.activate([
+            button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -24),
+            button.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            button.widthAnchor.constraint(equalToConstant: 200),
+            button.heightAnchor.constraint(equalToConstant: 40)
+        ])
     }
 
-    func syncPuckAndButton() {
-        // Update puck config
-        var configuration = Puck2DConfiguration(topImage: UIImage(named: "star"))
-        configuration.showsAccuracyRing = showsAccuracyRing
-        configuration.accuracyRingColor = UIColor.skyBlue
-        configuration.accuracyRingBorderColor = UIColor.lightGray
+    @objc public func changePuckOptions(sender: UIButton) {
+        let alert = UIAlertController(title: "Toggle Puck Options",
+                                      message: "Select an options to toggle.",
+                                      preferredStyle: .actionSheet)
+
+        alert.addAction(UIAlertAction(title: "Toggle Puck visibility", style: .default, handler: { _ in
+            self.showsPuck.toggle()
+        }))
```

Comment:
Nit: We do usually prefer the special syntax for trailing closure
```suggestion
        alert.addAction(UIAlertAction(title: "Toggle Puck visibility", style: .default) { _ in
            self.showsPuck.toggle()
        })
```